### ISSUE TITLE
VS Debugger/Azure Web Apps - application insights logger LogLevel is not set correctly

### ISSUE THREAD

**Issue by cijothomas** [microsoft/ApplicationInsights-aspnetcore#603]
If you are reporting bug/issue, please provide detailed Repro instructions.

## Repro Steps
1.  Create new Asp.Net Core Web App from VS default template. ( I picked WebApi)
2.  Configure AI ikey. (useai or addai extension method)
3. Start App from Visual Studio (debugging), fire requests

or
Just create a new app, make NO changes and publish to Azure WebApps, and enable AI in it. (via extensions)

##  Actual Behavior
1. All ILogger traces are sent to AI including a lot from aspnet core runtime. Example Traces:
![image](https://user-images.githubusercontent.com/5232798/36328574-710d4b68-1317-11e8-8cdb-8e45048fb385.png)

## Expected Behavior
No traces from runtime should be collected by default. This is likely unwanted cost to users. 
This occurs only when starting the app from VS Debugging/WebApps.

The key issue here is the 'default' level of logs sent to AI. It seems like all logs above 'information' are sent to AI. From my experiment, it looks like every Request will have atleast 5 Info traces from asp.net runtime itself. This is most likely polluting user experience and costing money.

## Version Info 
SDK Version : 2.2 or 2.1
.NET Version :	.NET CORE 2.0 						
How Application was onboarded with SDK(VisualStudio/StatusMonitor/Azure Extension) : VS
OS : Windows
Hosting Info (IIS/Azure WebApps/ etc) : Self-hosted locally

**Comment by cijothomas** [comment]
Modify appsettings.json like below to control applicationinsights logging behaviour.
"Logging": {
    "IncludeScopes": false,

    "ApplicationInsights": {
      "LogLevel": {
        "Default": "None"
      }
    },
    "LogLevel": {
      "Default": "Warning"
    }
  }

**Comment by cijothomas** [comment]
The default appsettings.json file for webapi in VS is:
{
  "Logging": {
    "IncludeScopes": false,
    "Debug": {
      "LogLevel": {
        "Default": "Warning"
      }
    },
    "Console": {
      "LogLevel": {
        "Default": "Warning"
      }
    }
  }
}
There is not default log level specified, so I believe for webapi anything >=Info is sent to AI.

**Comment by cijothomas** [comment]
```
{
  "Logging": {
    "IncludeScopes": false,
    "LogLevel": {
      "Default": "Warning"
    }
  }
}
```
This is the appsettings.json in webapplication template. This has a default level as Warning and only Warning or above is sent to AI.

**Comment by cijothomas** [comment]
Resolved with #853 . No auto-ilogger in debug mode. ilogger does only Warning or above in every case by default.
